features/users/profile_update.feature

```diff
@@ -0,0 +1,31 @@
+Feature: Update user profile
+
+In order to update my information
+As a learner
+I want to be able to edit my profile
+
+Background: Editing data to the database
+
+Scenario: Getting the profile page
```

Comment:
This scenario seems redundant, we could remove this and clean up the second scenarios steps. See my next comment 😄